src/account.ts

```diff
@@ -0,0 +1,32 @@
+import { BIP32Interface } from "bip32";
+import { networks, payments, crypto } from "liquidjs-lib";
+
+export default class Account {
+  node: BIP32Interface;
+  cache: Record<string, string>;
+
+  static BASE_DERIVATION_PATH = "m/84'/1776'/0'";
+  static BASE_DERIVATION_PATH_LEGACY = "m/84'/0'/0'";
+  static BASE_DERIVATIONT_PATH_TESTNET = "m/84'/1'/0'";
+
+
+  constructor(node: BIP32Interface, private network: networks.Network, private baseDerivationPath: string = Account.BASE_DERIVATION_PATH_LEGACY) {
+    this.node = node.derivePath(baseDerivationPath);
+    this.cache = {};
+  }
+
+  // Derive a range from start to end index of public keys applying the base derivation path
```

Comment:
this should be based on [this](https://github.com/Blockstream/gdk/blob/9fbc7abbea98f33f698b9be9da10a452860a7b94/subprojects/gdk_rust/gdk_electrum/src/account.rs#L867-L888), but I don't fully understand the code, especially they multiply start * BATCH_SIZE etc.. therefore I changed a bit

Pretty sure there is a reason, as not all the code as been ported